### ISSUE TITLE
Set params of only one layer

### ISSUE THREAD

**Issue by MarcoCiccone** [Lasagne/Lasagne#647]
Hi, 
I successfully used `set_all_param_values` function to reload the pretrained parameters of VGG16.
Now I want to add  BatchNorm after each conv layer so I need to load the parameters of the network 2 at the time (W and b) for each layer , and then apply batch_norm function.

Is it possibile, if I use `set_all_param_values` it try to set the parameters of all the layers below, but I need to to it just for a specific layer. Or maybe there is a better way..

Thank you for your help

**Comment by benanne** [comment]
You can use the `layer.get_params(**tags)` method to get the parameter variables associated with only the layer in question. You can then just call `set_value()` on each of them to give them new values.

**Comment by MarcoCiccone** [comment]
Sorry for the mistake and thank you for your help! I'll close the issue
